src/main/java/com/solvd/utilis/enums/Sorter.java

```diff
@@ -0,0 +1,9 @@
+package com.solvd.utilis.enums;
+
+public enum Sorter {
+
+    A_TO_Z,
```

Comment:
you can create a String parameter for each sort option and in this case you will be able to use it as a parameter for your locator
```suggestion
    A_TO_Z("Name (A to Z)"),
```